src/Agent.Worker/ContainerOperationProvider.cs

```diff
@@ -708,7 +708,14 @@ private async Task StartContainerAsync(IExecutionContext executionContext, Conta
                     Func<string, string, string, string> addUserWithIdAndGroup;
                     Func<string, string, string> addUserToGroup;
 
-                    if (isAlpineBasedImage)
+                    bool userIdIsLarge = Int64.Parse(container.CurrentUserId) > 256000;
```

Comment:
Do we always expect `CurrentUserId` will be a valid `int`?

Reply:
Yes, user ID should be a valid `int`